ledger/sandbox/BUILD.bazel

```diff
@@ -395,7 +395,7 @@ server_conformance_test(
 server_conformance_test(
     name = "conformance-test-config-management",
     server_args = [
-        "--static-time",
+        "--wall-clock-time",
```

Comment:
Can we change `"conformance-test-lots-of-parties"` too?

Reply:
Done.